app/retail/templates/emails/new_kudos.html

```diff
@@ -51,81 +50,118 @@
+    letter-spacing: 0;
+    text-align: center;
+    line-height: 59px;
+  }
+  .kudos-fancy-bg h3 {
+    color: #FFFFFF;
+    letter-spacing: 0;
+    text-align: center;
+    line-height: 50px;
+  }
+  .kudos-fancy-bg p {
+    color: #FFFFFF;
+  }
+  .kudos-fancy-bg .kudos-sparkle {
+    position: absolute;
+    background-image: url('data:image/svg+xml;utf8,<svg width="185px" height="178px" viewBox="0 0 185 178" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">        <g id="Received-Kudos" transform="translate(-692.000000, -1019.000000)" fill="#FFFFFF">            <g id="sparkle1" transform="translate(692.000000, 1019.000000)">                <path d="M42.2738534, 101.302405 C42.2738534, 101.302405 52.8812744, 46.6219931 62.4279533, 101.302405 C62.4279533, 101.302405 115.465058, 110.766323 62.4279533, 120.230241 C62.4279533, 120.230241 52.8812744, 175.962199 42.2738534, 120.230241 C10.4515903, 114.446735 12.0427035, 108.137457 42.2738534, 101.302405 Z" id="Path"></path>                <path d="M14.5670485, 14.9749304 C14.5670485, 14.9749304 20.9523044, -18.718663 27.3375604, 14.9749304 C27.3375604, 14.9749304 60.3280495, 20.7660167 27.3375604, 26.5571031 C27.3375604, 26.5571031 21.4844091, 61.3036212 14.5670485, 26.5571031 C-5.1208241, 22.8718663 -4.58871943, 18.6601671 14.5670485, 14.9749304 Z" id="Path"></path>                <path d="M157.567048, 150.97493 C157.567048, 150.97493 163.952304, 117.281337 170.33756, 150.97493 C170.33756, 150.97493 203.32805, 156.766017 170.33756, 162.557103 C170.33756, 162.557103 164.484409, 197.303621 157.567048, 162.557103 C137.879176, 158.871866 138.411281, 154.660167 157.567048, 150.97493 Z" id="Path"></path>            </g>        </g>    </g></svg>');
+    width: 185px;
+    height: 178px;
+    top: 45%;
+    left: 25%;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: 70px;
+  }
+  .kudos-fancy-bg .kudos-sparkle-reverse {
+    transform: scaleX(-1) scaleY(-1);
+    left: 50%;
+    top: -20%;
+  }
+  .kudos-fancy-bg .kudos-hex-container {
+    position: relative;
+    background-image: url('data:image/svg+xml;utf8,<svg viewBox="0 0 329 382" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">        <g id="Received-Kudos" transform="translate(-875.000000, -799.000000)" stroke="#FFFFFF" stroke-width="20">            <g id="Group-5-Copy-7" transform="translate(1039.124954, 989.889382) rotate(-330.000000) translate(-1039.124954, -989.889382) translate(862.124954, 833.389382)">                <polygon id="Stroke-2" points="265.627611 0.408274735 89.1276108 0.408274735 0.880108514 156.410824 89.1276108 312.408275 265.627611 312.408275 353.880109 156.410824"></polygon>            </g>        </g>    </g></svg>');
```

Comment:
My recommendation is don't use svg in email https://css-tricks.com/a-guide-on-svg-support-in-email/ here is a list of svg compatibility and techniques